src/restore_db.py

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python3
+
+import database
+import backup
+from service import service
+from settings import get_settings
+from os.path import abspath, dirname
+from os import chdir
+
+def restore_db():
+    ok = service.status == 'running' and service.volume_present
+    if not ok:
+        raise Exception('montagu not in a state we can restore')
```

Comment:
It would be good if this could print out the current services status and volume present values, so that we can debug from looking at logs later.

Reply:
I've raised this as https://vimc.myjetbrains.com/youtrack/issue/VIMC-957 since the PR is already merged.